src/node/node_class.hpp

```diff
@@ -284,7 +287,7 @@ inline auto& ObjectWrap<ClassType>::get_nativeMethods() {
  
 template<typename ClassType>
 inline auto& ObjectWrap<ClassType>::get_schemaObjectTypes() {
-	static std::unordered_map<std::string, std::unordered_map<std::string, SchemaObjectType*>*> s_schemaObjectTypes;
+	static std::unordered_map<std::string, std::unordered_map<std::string, SchemaObjectType>> s_schemaObjectTypes;
```

Comment:
While I was in here, I refactored the code a bit to reflect some of the simplifications I did in the hermes port. I tried to reduce a lot of the duplicated logic (eg there were 4 different places where we were constructing the object)